doc/source/whatsnew/v0.25.0.rst

```diff
@@ -28,6 +28,25 @@ Backwards incompatible API changes
 
 .. _whatsnew_0250.api.other:
 
+Indexing and getitem accept zerodim np.ndarray
```

Comment:
on 2nd thought, just add a single line whatsnew entry in Other Enhancements

Reply:
OK!